nuxeo-features/rest-api/nuxeo-rest-api-server/src/main/java/org/nuxeo/ecm/restapi/server/jaxrs/BulkActionObject.java

```diff
@@ -76,11 +83,14 @@ public Response executeBulkAction(@PathParam("actionId") String actionId, String
         String repository = getContext().getCoreSession().getRepositoryName();
         String username = getContext().getPrincipal().getName();
 
-        BulkCommand command = new BulkCommand.Builder(actionId, query).repository(repository)
-                                                                      .user(username)
-                                                                      .params(BulkParameters.paramsToMap(actionParams))
-                                                                      .scroller(scroller)
-                                                                      .build();
+        Builder builder = new BulkCommand.Builder(actionId, query).repository(repository)
+                                                                  .user(username)
+                                                                  .params(BulkParameters.paramsToMap(actionParams))
+                                                                  .scroller(scroller);
+        if (queryLimit > 0) {
+            builder.queryLimit(queryLimit);
+        }
+        BulkCommand command = builder.build();
```

Comment:
Is this necessary since `queryLimit` is marked as nullable in `BulkCommand`? I suspect that we can simply use the fluent constructor as for the scroller.

Reply:
OK the construcotr throws if <= 0.